src/utils/EventUtils.hpp

```diff
@@ -121,84 +46,63 @@ class EventRegistry {
   /// Sets the global start time
   /**
    * @param[in] applicationName A name that is added to the logfile to distinguish different participants
-   * @param[in] runName A name of the run, will be printed as a separate column with each Event.
-   * @param[in] comm MPI communicator which is used for barriers and collecting information from ranks.
+   * @param[in] filePrefix A prefix for the file name.
```

Comment:
Previously, the `applicationName` was `precice-ParticipantName` and the `runName` was generally unused.

I decided to split this up, so that I can use the participant name internally.

`applicationName` is now `SolverName` and the file prefix is `precice`.